Source/Scene/Tileset3DTileContentProvider.js

```diff
@@ -12,11 +12,41 @@ define([
     /**
      * @private
      */
-    var Tileset3DTileContentProvider = function() {
+    var Tileset3DTileContentProvider = function(tileset, tile, url) {
```

Comment:
Throughout this branch, in a separate PR, can you change the function declarations to, for example,
```javascript
function Tileset3DTileContentProvider(tileset, tile, url) {
```
See #3329